python/pyarrow/parquet.py

```diff
@@ -988,7 +967,9 @@ def __init__(self, path_or_paths, filesystem=None, schema=None,
 
         if metadata is None and self.metadata_path is not None:
             with self.fs.open(self.metadata_path) as f:
-                self.metadata = ParquetFile(f, memory_map=memory_map).metadata
+                self.metadata = ParquetFile(
+                    f, memory_map=memory_map,
+                    read_dictionary=read_dictionary).metadata
```

Comment:
Does the `read_dictionary` setting influence the `metadata` ?

Reply:
no, I can remove this. It does change the Arrow schema though

Reply:
done